pkg/okteto/client_test.go

```diff
@@ -25,6 +26,81 @@ import (
 	"golang.org/x/oauth2"
 )
 
+type fakeGraphQLClient struct {
```

Comment:
Shouldn't these structs go in `action_test.go`?

Reply:
The plan is that `fakeGraphQLClient` is reused across all the files and not only actions, i.e: `secrets_test.go`, `auth_test.go`...